dockstore-common/src/main/java/io/dockstore/common/yaml/AbstractYamlService.java

```diff
@@ -158,6 +161,15 @@ public void setLatestTagAsDefault(boolean latestTagAsDefault) {
         this.latestTagAsDefault = latestTagAsDefault;
     }
 
+    @AssertTrue
```

Comment:
Can you javadoc all these places that can be removed when we turn automatic DOI across the board with the same searchable comment, could be e.g.
`//  SEAB-6805]`